src/main/java/Entities/Event.java

```diff
@@ -1,48 +0,0 @@
-package Entities;
```

Comment:
this almost certainly should not have been deleted

Reply:
or, if this had to be deleted, there should be other events and event subclasses in the Entities package because according to clean architecture that's where they belong

Reply:
I'll move it to the Entities package